architecture/core_concepts/routes.adoc

```diff
@@ -332,3 +332,95 @@ certificate to validate the endpoint certificate, securing the connection
 from the router to the destination.
 
 ====
+
+== Route Hostnames
+In order for services to be exposed externally, an OpenShift route allows
+you to associate a service with an externally-reachable hostname.
+This edge hostname is then used to route traffic to the service.
+
+.An Route with a specified host:
+====
```

Comment:
Should the above and below example headings be "A Route with a Specified Host"? 

s/a/an